### DISCUSSION TITLE
Unlock Windows Laptops

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#273] [Q&A]
Hi guys, is there a way to unlock windows laptops with GTS 4 Mini? For Mi Band exists an App that let you do it.

**Comment by silver-zepp** [comment]
Hey @Cotrox I'm afraid there's no official way to unlock windows with one of our devices. I would recommend asking the developer of the app you've mentioned if they could implement it for a broader range of devices. Also, you can search for alternatives to https://github.com/tiktaalik-dev/blueproximity. Maybe some of them will work on your machine.
